src/app/modules/practitioner-registration/pages/practitioner-attachment/practitioner-attachment.component.ts

```diff
@@ -0,0 +1,160 @@
+import { Component, OnInit } from '@angular/core';
+import { RegistrationForm } from '../../models/registration-form';
+import { PRACTITIONER_REGISTRATION_PAGES } from '../../practitioner-registration-route-constants';
+import { RegistrationContainerService } from '../../services/registration-container.service';
+import { Router } from '@angular/router';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { CreatePractitionerDataService } from '../../services/create-practitioner-data.service';
+import { PRACTITIONER_ATTACHMENT } from '../../models/practitioner-attachment';
+
+interface RadioItem {
```

Comment:
IRadioItems <= Interface for radio buttons labels, you do not need to add RadioItem.

Reply:
Done.